snippets/mapped-table.md

```diff
@@ -0,0 +1,35 @@
+### MappedTable
+
+Generate a table with dynamic rows based on passed array of data. Optionally include headers.
+
+Conditionaly check weather to include table headers or not. Use `Array.prototype.map` to map over array of data and return `<tr>` for each entry.
+
+```jsx
+function MappedTable(props) {
+  return (
+    <tbody>
+      {props.headers ? return(<tr>{props.headers.map((h, i)=> (<th key={i}>h</th>))}</tr>) : return null}
+      {props.data.map((v, i) => {
+        return (
+          <tr obj={v} key={v.id ? v.id : i}>
+            <td>v.name</td>
+          </tr>
+        );
+      })}
+    </tbody>
```

Comment:
Technically speaking, if we want to be very strict about semantics, we should allow users to pass `props.footers` to actually add a footer to the table and `props.caption` to caption it. If that is necessary or if it's a common use-case remains to be seen, I'm just stating this to be on the record for further discussion.